libcaf_test/caf/test/runner.cpp

```diff
@@ -53,6 +53,24 @@ std::optional<unsigned> parse_log_level(std::string_view x) {
   return {};
 }
 
+std::optional<std::regex> to_regex(std::string_view regex_string) {
+#ifdef CAF_ENABLE_EXCEPTIONS
+  try {
+    return std::regex{regex_string.begin(), regex_string.end()};
+  } catch (std::regex_error& err) {
```

Comment:
Minor nitpick, this can be const and we can generalize it to any standard exception:

```suggestion
  } catch (const std::exception& err) {
```